docs/logentry/processing-log-lines.md

```diff
@@ -0,0 +1,497 @@
+# Processing Log Lines
```

Comment:
I think the doc has moved since then, that's why you're getting this addition no ?